src/templates/app.vue

```diff
@@ -25,48 +25,54 @@ limitations under the License.
-            <DODNS :loaded="data !== ''" :data="data"></DODNS>
-            <RecordBase ref="RecordBase" :data="data" :registrar="registrar"></RecordBase>
-        </div>
-        <footer class="footer" style="align-self: flex-end; padding: 20px; width: 100%;">
-            <div class="content has-text-centered">
-                <p>
-                    <a href="#top">Back to Top</a>
-                </p>
-                <p>
-                    Thanks to <a href="https://cloudflare.com">Cloudflare</a> for their great WHOIS/DNS-over-HTTPS APIs.
-                    You can learn more about the importance of DNS-over-HTTPS and how to use it
-                    <a href="https://developers.cloudflare.com/1.1.1.1/dns-over-https/">here.</a>
-                </p>
-                <p>
-                    Thanks to <a href="https://twitter.com/matthewgall">Matthew Gall</a> for his wonderful
-                    <a href="https://whoisjs.com/">WHOIS API.</a>
-                </p>
+    <div>
+        <GHLink colour="0069ff" repo="https://github.com/do-community/dns-tool"></GHLink>     
+        <div class="container" style="display: flex; flex-direction: column;">
+            <RecordSelectionModal ref="RecordSelectionModal"></RecordSelectionModal>
+            <div id="top" class="has-text-centered" style="padding-left: 30%; padding-right: 30%; margin-top: 10px">
+                <h2 class="title is-2">
+                    DNS Lookup Tool
+                </h2>
+                <h5 class="title is-5">
+                    Enter the (sub-)domain you wish to look up.
+                </h5>
+                <span v-if="data !== ''">
+                    <hr style="margin: 10px">
+                    <p><a @click="toggleRecordTextModal">Get the records in text/markdown form.</a></p>
+                </span>
+                <form autocomplete="on" style="margin-top: 20px" @submit.prevent="searchDNSEvent">
+                    <input id="DomainInput" class="input" type="text" placeholder="Domain">
+                    <button id="SearchButton" class="button is-link" style="margin-top: 20px">
+                        Search DNS Records
+                    </button>
+                </form>
             </div>
-        </footer>
+            <hr>
+            <div id="content">
+                <RecordJumps :loaded="data !== ''"></RecordJumps>
+                <DODNS :loaded="data !== ''" :data="data"></DODNS>
+                <RecordBase ref="RecordBase" :data="data" :registrar="registrar"></RecordBase>
+            </div>
+            <footer class="footer" style="align-self: flex-end; padding: 20px; width: 100%;">
+                <div class="content has-text-centered">
+                    <p>
+                        <a href="#top">Back to Top</a>
+                    </p>
+                    <p>
+                        This tool is open source on GitHub under the Apache-2.0 license! We welcome feedback and contributions.
```

Comment:
`open source on GitHub` -> `open-source on GitHub` & link to repo
`Apache-2.0 license` link to license file